index.markdown

```diff
@@ -101,6 +101,14 @@ title="ro-crate-java"
 subtitle="A library to package and link all your research data"
 content="Using ro-crate-java, you can create and modify research packages (RO-Crates). These packages allow you to describe your research machine-readable and human-readable, making it FAIR. Build this functionality into your applications or use it directly."
 %}
+
+{% include card.html tags="engineering data repositories"
+target="https://rxp.datamanager.kit.edu"
```

Comment:
Did you discuss with Rainer or Thomas if we plan to directly link to other pages? The other ones do have an extra page with documentation etc.